src/app/components/confirm-signup/confirm-signup.component.html

```diff
@@ -1,7 +1,23 @@
-<form (ngSubmit)="onSubmit(confirmForm)" #confirmForm="ngForm">
-    <label for="email">Email:</label>
-    <input type="email" id="email" name="email" ngModel required />
-    <label for="code">Confirmation Code:</label>
-    <input type="text" id="code" name="code" ngModel required />
-    <button type="submit">Confirm Sign Up</button>
-</form>
\ No newline at end of file
+<div class="cover">
+    <section class="hero-image">
+    </section>
+    <section class="form-component">
+        <section class="logo">
+            <img src="../../assets/logo.svg" alt="logo" />
+            <h1>FREELO</h1>
+        </section>
+        <form (ngSubmit)="onSubmit(confirmForm)" #confirmForm="ngForm">
+            <mat-form-field color="accent" class="full-width">
+                <mat-label for="email">Email</mat-label>
+                <input type="email" id="email" matInput name="email" ngModel required />
+                <mat-error *ngIf="confirmForm.controls.email.invalid">Please enter a valid email</mat-error>
+            </mat-form-field>
+            <mat-form-field color="accent" class="full-width">
+                <mat-label for="code">Confirmation Code</mat-label>
+                <input type="text" id="code" matInput name="code" ngModel required />
+                <mat-error *ngIf="confirmForm.controls.code.invalid">Please enter a valid code</mat-error>
```

Comment:
Please use @if instead of ngif

Reply:
Like this? `*@if="signupForm.controls.email.invalid"`

Reply:
https://blog.angular-university.io/angular-if/